### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Added Video Listing Page with routing.

### CODE REVIEW

File: src/actions/videoAction.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const getVideos = () => async (videoDispatch) => {
+    try {
+        const { data } = await axios.get('/api/videos');
+        videoDispatch({type: 'SUCCESS', payload: data.videos});
```
Comment (sameep-s) [inline src/actions/videoAction.js:6]: One suggestion use can use payload = {videos: data.videos} ie you can make payload an object as it is a good practice if you want to give multiple values with one single payload.

File: src/Screen/VideoListing/VideoListing.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Card from '../../components/Card/Card';
+import SideBar from '../../components/SideBar/SideBar';
+import { useVideo } from '../../context/index'
+import './VideoListing.css'
+
+const VideoListing = () => {
+    const { videoState: {loading, error, videos} } = useVideo();
+    console.log(videos, loading, error);
```
Comment (sameep-s) [inline src/Screen/VideoListing/VideoListing.jsx:8]: You can remove these console.log's

**Review by sameep-s — APPROVED** [review]
I have written some comments just take a look at them and you are fine.

File: src/actions/videoAction.js (hunk 1 above)
Comment (munjalSaksham71) [reply src/actions/videoAction.js:6]: Here, I am just getting videos only - I Guess there is no need to do it. Maybe use in future cases and thanks for suggesting this way.

File: src/components/Card/Card.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.video_card {
+    width: 17rem;
+    gap: 10px;
```
Comment (rushikesh3299) [inline src/components/Card/Card.css:3]: good practice that you used px for smaller values

**Review by rushikesh3299 — APPROVED** [review]
Looks good to me

**Pull request merged**